lib/debug.py

```diff
@@ -175,7 +276,7 @@ def init_svg(self):
 
     def save_svg(self):
         tree = etree.ElementTree(self.svg)
-        debug_svg = os.path.join(os.path.dirname(os.path.dirname(__file__)), "debug.svg")
+        debug_svg = self.debug_dir / self.debug_svg_file
```

Comment:
Thank you for your work on this so far.
Can you change this to `debug_svg = str(self.debug_dir / self.debug_svg_file)` ?
Otherwise I'll get an TypeError: File or filename expected, got 'PosixPath'

Reply:
I didn't realize that as of lxml version 5.0.0, it supports the pathlib Path.